RealifeTech/SDKFactory.swift

```diff
@@ -50,6 +50,5 @@ public class RealifeTech {
         }
         Communicate = CommunicateFactory.makeCommunicateModule()
         Content = ContentFactory.makeContentModule(graphQLDispatcher: dispatcher)
-        CoreFactory.requestValidToken(fromApiHelper: apiHelper) { }
```

Comment:
The reason for removing this from `ConfigureSDK` is because this would be handled in the device registration. I believe the user is aware of calling device registration before using SDK as there's a line in the README `Interfacing with our backend systems requires that your device be registered with them.`

Reply:
Just to confirm, the SDK should handle device registration right?
```
RealifeTech.General().registerDevice() -> Void
RealifeTech.General().sdkReady: Bool
RealifeTech.General().deviceIdentifier: String (read only, returns our device id)
```

Reply:
I think so as the backend service has to be attached the device ID.
If the app uses other interfaces to make the API call, the SDK also pre-dispatch the `getValidToken` from the APIRequester.